web/app.js

```diff
@@ -400,6 +392,13 @@ const PDFViewerApplication = {
     this.mlManager?.setEventBus(eventBus, this._globalAbortController.signal);
 
     this.overlayManager = new OverlayManager();
+    this._touchManager = new TouchManager({
+      container: window,
+      overlayManager: this.overlayManager,
+      onPinching: this.touchPinchCallback.bind(this),
+      onPinchEnd: this.touchPinchEndCallback.bind(this),
+      signal: this._windowAbortController.signal,
+    });
```

Comment:
Sorry, but why can't we just initialize the `TouchManager` in the `bindWindowEvents`-method instead?

To me it seems like it'd "fix" a couple of issues with the current implementation (that I've not commented about, hoping that we can just move the code instead).